packages/site-content/package.json

```diff
@@ -18,31 +19,48 @@
         "build:dev"
       ]
     },
+    "build:config": {
+      "command": "tsc --pretty",
+      "files": [
+        "src/",
+        "tsconfig.json",
+        "../../tsconfig.base.json"
+      ],
+      "output": [
+        "lib/",
+        "tsconfig.tsbuildinfo"
+      ],
+      "dependencies": [
+        "../site-templates:build:types",
+        "../site-client:build:types"
+      ],
+      "clean": "if-file-deleted"
+    },
     "build:prod": {
-      "command": "eleventy --config=eleventy.config.prod.js",
+      "command": "eleventy --config=lib/eleventy.config.prod.js",
```

Comment:
I don't mind doing outDir='.' and gitignores to keep this file at the top level, fwiw.

Reply:
Done